Makefile

```diff
@@ -1,16 +1,22 @@
+# THOUGHT: assume this makefile is for local builds and ci would go directly into modules if we want to separate image dependecies
+# eg has Java but no Go? I think this + avoiding unnecessary rebuilds at module levels could allow us to simplify our make structure
+# and speed up builds too
+
```

Comment:
this could probably let us do away with the separation between get-deps and build at this makefile level if we want to. but it doesn't seem worth the effort 🤔

Reply:
Yeah, this is fine for now. Don't forget to remove this comment. :slightly_smiling_face: